lib/gitlab/ci/templates/pipeline/GradleInstall4JPipeline.yml

```diff
@@ -30,7 +31,7 @@ before_script:
   variables:
     SNAPSHOT: "-Psnapshot" # default to snapshot
   artifacts:
-    name: installers
+    name: $INSTALLER_NAME installers
     expire_in: 10 days
```

Comment:
I'm open to re-arranging the installer name here as I'm unsure if INSTALL_NAME is blank, then there will be an extra white space character that may mess with people who use command line tools.

Reply:
Can you assign INSTALLER_NAME the default value of "installers" in the variables section at the top of the file and add a description so it's easier for folks to override the value in manual pipelines?

`
INSTALLER_NAME:
    value: "installers"
    description: "The name of the installer artifacts that can be downloaded after the job completes"
`

Reply:
Should be done :)